ui/src/app/intro/views/MaasIntroSuccess/MaasIntroSuccess.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button, Card, Icon, List } from "@canonical/react-components";
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+
+import Section from "app/base/components/Section";
+import { useWindowTitle } from "app/base/hooks";
+import dashboardURLs from "app/dashboard/urls";
+import introURLs from "app/intro/urls";
+import machineURLs from "app/machines/urls";
+import authSelectors from "app/store/auth/selectors";
+import { actions as configActions } from "app/store/config";
+
+const MaasIntroSuccess = (): JSX.Element => {
+  const dispatch = useDispatch();
+  const authUser = useSelector(authSelectors.get);
+  useWindowTitle("Welcome - Success");
+
+  let continueDetails = {
+    label: "Set up SSH keys",
+    link: introURLs.user,
+  };
+
+  if (authUser?.completed_intro) {
+    if (authUser?.is_superuser) {
+      continueDetails = {
+        label: "Go to dashboard",
```

Comment:
I wonder if these labels should be more obviously something the user needs to click to finish the setup, maybe they could be "Finish setup" or "Complete setup".

Either that or we could make it so it dispatches the config update when the user gets to the screen.

Reply:
I like "Finish setup" so I might go with that